common/src/main/java/com/wynntils/core/features/properties/FeatureInfo.java

```diff
@@ -20,4 +20,37 @@
 
     /** Subjective Performance impact of feature */
     PerformanceImpact performance() default PerformanceImpact.MEDIUM;
+
+    /**
+     * How stable a feature is to wynncraft changes, how able it is to be maintained, and other similar
+     * judgements. Subjective.
+     */
+    enum Stability {
+        EXPERIMENTAL,
+        UNSTABLE,
+        STABLE,
+        INVARIABLE
+    }
+
+    /**
+     * How much impact on gameplay a feature has. For example, a more important feature like a minimap
+     * would have a larger impact than a smaller feature like blocking health pots. Subjective.
+     */
+    enum GameplayImpact {
+        SMALL,
+        MEDIUM,
+        LARGE,
+        EXTREME
+    }
+
+    /**
+     * How much performance strain the feature causes. For example, cached chunks would have a higher
+     * performance impact than a Wynncraft Button. Subjective.
+     */
+    enum PerformanceImpact {
+        SMALL,
+        MEDIUM,
+        LARGE,
+        EXTREME
+    }
```

Comment:
In all seriousness, how much is this helping us, or the user? Have we ever had unstable features? Will we ever release any? If a feature causes a crash, we'd likely only even know that after the code has been released, and it will be called a bug, and fixed.

The gameplay impact is surely just subjective, and can easily be discovered by the user. For me, things like bank search is a huge boon, but it might look like a "small" feature to anyone else.

And finally, performance impact. In theory, this could be measured. Are you going to do that? If not, it's just a subjective guess. Apart from perhaps heyzeer0s fireflies rendering stuff, i seriously doubt any of our features has anything but negligible performance impact.

I've been annoyed by this before. And we do not need to discuss this in this PR. But I think we should drop these annotataions. And now might be a good time.

Reply:
These annotations are sure subjective. I would agree that what we have currently is not useful. Though, I am very interested in having the whole `FeatureInfo` annotation. 
Few of my ideas for hopefully more useful fields (for example, the user could use these in the config menu, where all features are listed):

- First release version of feature
- Last patched (use this and first release to indicate that something has changed?)
- Having CE on Artemis, we should try to aim to actually release "beta" features, having a flag for that. (this also meaning that CE is actually a beta, and release stream actually gets patched every week or in any reasonable time periods)
- Search flags/aliases (aliases are just alternative search terms, flags could be "GUI", "Bank", "Commands", "Overlay", "Guilds", "Quests")

Just some of my ideas, needs a bit of refining. :)

Reply:
Yeah these annotations were originally planned by me to give information on the feature that is relevant. I’m fine with performance/gameplay impact being dropped, but I think stability should be kept: i.e. dialogue parsing has historically broken multiple times. It also is imo similar to cutting edge vs stable in a way

Reply:
Suggested compromise: drop gameplay/performance, and keep stability for now. Set it at "stable" initially but drop it by one level every time we discover a bug in the feature. 😈  (half-serious, half-joking now)

Reply:
Yeah something like that

Reply:
feeling a bit mischievous, might release an "unstable" feature that purposely crashes the game

Reply:
@P0keDev Can you remove gameplay/performance annotations?